builder/__main__.py

```diff
@@ -0,0 +1,1500 @@
+# Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This used to be builder.py

Reply:
I have never seen a `__main__.py` file before
this might be python3-only?
I don't personally care if it breaks python2, we could always rename it later and its usage would stay the same